### PR TITLE
Feature/spring rest

### PR DESCRIPTION
**Pull request by Bakhodirov-Jakhongir**

### CODE REVIEW

File: src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAO.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import com.jakhongir.springboot.demo.mycoolapp.entity.Employee;
+
+public interface EmployeeDAO {
+    List<Employee> getAll();
+
+    Employee findById(int id);
+
+    Employee save(Employee employee);
+
+    Employee updateOrCreate(Employee employee);
```
Comment (Jakhongirkhoja-Bakhodirov) [inline src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAO.java:14]: Are you using this method

File: src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAOImple.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    @Autowired
+    public EmployeeDAOImple(EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    @Override
+    public List<Employee> getAll() {
+        System.out.println("Employee DAO Imple");
+        TypedQuery<Employee> employees = this.entityManager.createQuery("FROM Employee order by id", Employee.class);
+      //  System.out.println(employees.getResultList());
```
Comment (Jakhongirkhoja-Bakhodirov) [inline src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAOImple.java:26]: You can remove any extra comments

**Review by Jakhongirkhoja-Bakhodirov — COMMENTED** [review]
You can use build in Spring Boot Data JPA for CRUD in order to avoid extra codes and files

File: src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAO.java (hunk 1 above)
Comment (Bakhodirov-Jakhongir) [reply src/main/java/com/jakhongir/springboot/demo/mycoolapp/dao/EmployeeDAO.java:14]: I will remove

**Pull request merged**